index.bs

```diff
@@ -822,6 +818,20 @@ When an {{XRReferenceSpace}} is requested, the user agent MUST <dfn>create a ref
 
 </div>
 
+<div class="algorithm" data-algorithm="get-offset-space">
+The <dfn method for="XRReferenceSpace">getOffsetReferenceSpace(|originOffset|)</dfn> method MUST perform the following steps when invoked:
+
+  1. Let |base| be the {{XRReferenceSpace}} the method was called on.
+  1. If |base| is an instance of {{XRStationaryReferenceSpace}}, let |offsetSpace| be a new {{XRStationaryReferenceSpace}}.
```

Comment:
Are you planning to rebase on top of #587?

Reply:
Comments in https://github.com/immersive-web/webxr/pull/587#discussion_r279622789 may indicate that it's better to land this one first?